sale_payment_method/README.rst

```diff
@@ -0,0 +1,61 @@
+Sale Payment Method
```

Comment:
Please add above another row of equal signs, see https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst as reference.